source/renderer/app/domains/News.js

```diff
@@ -31,6 +31,7 @@ export type NewsTypesStateType = {
 const { version, platform } = global.environment;
 
 class News {
+  @observable id: number;
```

Comment:
Not really important at the moment, but this is probably going to be a string, won't it?

Reply:
@daniloprates yeah can be but now I imagine that like number, like timestamp was before as identification. @darko-mijic what is maybe best from your perspective as `id` in JSON file at this point, a string or number?

Reply:
We were discussing this and concluded that we can continue using timestamps (numbers) as IDs. We will know when the newsfeed was updated and we get uniqueness. There is some room for mistakes when copy-pasting old news when creating new. But that issue is present with any kind of id since newsfeed updates are currently a manual process. 
We are good with this for now.